synapse/lib/stormsvc.py

```diff
@@ -125,6 +130,22 @@ class StormSvcClient(s_base.Base, s_stormtypes.StormType):
         if 'StormSvc' in names:
 
             self.info = await proxy.getStormSvcInfo()
+            try:
+                init = self.info.get('init', '')
```

Comment:
it's small/subtle, but lets not default this ( or fini ) to `''` and make the check below for `is None` for consistency with our other default value checks.